Automattic/wp-calypso
# Framework: downgrade to react-virtualized 9.9.0

We recently upgraded from react-virtualized 9.4.0 to 9.10.1, particularly for some accessibility improvements offered in the latest release. Unfortunately, this has led to some wonky behaviour of Reader's Manage Following, documented in https://github.com/Automattic/wp-calypso/issues/18387.

Downgrading to 9.9.0 appears to fix the problems we're experiencing.

Fixes https://github.com/Automattic/wp-calypso/issues/18387.

### To test

Head to http://calypso.localhost:3000/following/manage and perform a search of your existing feeds. Ensure the the search results are formatted correctly. 

Compare with wpcalypso, running 9.10.1 - you sometimes may see the search results appear well down the page instead of under the search box.
